### PR TITLE
support dropping kms-keys

### PR DESCRIPTION
**Pull request by ideahitme**
/cc @tuxlife @mikkeloscar 

Support dropping kms-key, added TODO as well

### CODE REVIEW

File: sevenseconds/config/kms.py [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 import json
 
 
+# TODO:support reverting Drop:true operation by either cancelling deletion or recreating the keys
 def configure_kms_keys(account: object, region):
     keys_config = account.config.get('kms', {})
     kms_client = account.session.client('kms', region)
     for key_alias in keys_config:
         key_config = keys_config[key_alias]
+        if 'Drop' in key_config and key_config['Drop']:
```
Comment (tuxlife) [inline sevenseconds/config/kms.py]: You can use this `if key_config.get('Drop', False):`

Comment (tuxlife) [reply sevenseconds/config/kms.py]: And please use lowercase config names.

**Review by tuxlife — CHANGES_REQUESTED** [review]

File: sevenseconds/config/kms.py [hunk 2]
```diff
@@ -52,3 +57,32 @@ def configure_kms_keys(account: object, region):
                         .format(key_alias, key_id, alias_response)
                     )
                     continue
+
+
+def schedule_key_deletion(kms_client, key_alias):
+    key_alias = "alias/foo-bar"
```
Comment (ideahitme) [inline sevenseconds/config/kms.py]: accident :(

**Comment by ideahitme** [comment]
@tuxlife changed, please look again

**Comment by tuxlife** [comment]
@ideahitme please replace the rest of the new keys with lowercase name with underscore

`KeyUsage -> key_usage`
`KeyPolicy -> key_policy`

**Comment by ideahitme** [comment]
@tuxlife done

**Review by tuxlife — APPROVED** [review]

**Comment by tuxlife** [comment]
:+1:

**Comment by ideahitme** [comment]
👍

**Pull request merged**